src/features/components/molecules/layout-overlay/cfd-warning-banner/cfd-warning-banner.module.scss

```diff
@@ -0,0 +1,3 @@
+.cfd_banner_container {
+    box-shadow: 0px -4px 10px rgba(65, 70, 82, 0.1);
+}
```

Comment:
instead of create new not mandatory file, we can pass this property via `style`
```suggestion
<Container.Fixed style={{ boxShadow: 0px -4px 10px rgba(65, 70, 82, 0.1) }} bgcolor="primary"> 
```

Reply:
we don't use inline styles, this is ok.